cmake/TransmuteCore.cmake

```diff
@@ -3,6 +3,7 @@ file(GLOB_RECURSE TR_CORE_SOURCE
     "src/runtime/*.cpp"
     "src/renderer/*.cpp"
     "src/xr/*.cpp"
+    "src/runtime/extension/*.cpp"
```

Comment:
No need to add this line, because these files has been included by "src/runtime/*.cpp" with `GLOB_RECURSE`.